src/javascript/binary/websocket_pages/user/new_account/virtual_acc_opening.js

```diff
@@ -106,13 +106,13 @@ const VirtualAccOpening = (() => {
             const residence   = response.echo_req.residence;
             Client.set('residence', residence, new_account.client_id);
             LocalStore.remove('gclid');
-            BinarySocket.send({ landing_company: residence }).then((response_lc) => {
+            BinarySocket.send({ landing_company: residence }).then(() => {
```

Comment:
if we don't need the response from landing company anymore can we just remove this websocket send and wait?